test/Microsoft.SqlTools.ServiceLayer.Test/LanguageServer/PeekDefinitionTests.cs

```diff
@@ -0,0 +1,228 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+#define LIVE_CONNECTION_TESTS
```

Comment:
We define this in one of the build configurations at compile-time, so this probably shouldn't be defined here.